src/Knp/Menu/MenuItem.php

```diff
@@ -24,21 +24,21 @@ class MenuItem implements ItemInterface
     /**
      * Attributes for the item link
      *
-     * @var array
+     * @var array<string, mixed>
      */
     protected $linkAttributes = [];
 
     /**
      * Attributes for the children list
      *
-     * @var array
+     * @var array<string, mixed>
```

Comment:
link attributes and children attributes also expect `string|bool|null` for values.

And the ItemInterface should be updated to also use this type instead of `mixed`